types/mongoose/index.d.ts

```diff
@@ -2435,6 +2435,7 @@ declare module "mongoose" {
      * If a callback is not passed, the aggregate itself is returned.
      * @param aggregations pipeline operator(s) or operator array
      */
+    aggregate(): Aggregate<any[]>;
     aggregate(aggregations: any[]): Aggregate<any[]>;
```

Comment:
Since the above two overloads each return an `Aggregate<any[]>` can you instead make `aggregations` optional?

```diff
- aggregate(): Aggregate<any[]>;
- aggregate(aggregations: any[]): Aggregate<any[]>;
+ aggregate(aggregations?: any[]): Aggregate<any[]>;
```